Adding SRW results (#21)
dtcenter/ufs-case-studies: docs/source/2018Michael.rst

    .. code-block:: bash
   
      crontab Michael.crontab     

>   .. group-tab:: SRW.v1.0

Reviewer: reached to 140 knots -> reached 140 knots

> 

Reviewer: made landfall in Florida (not at)

>     The case runs are initialized at 00z Oct 09, 2018 with 90 hours forecasting. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, cycle hour, and fcst length are listed below.

Reviewer: This change is also needed in the MRW section
```suggestion
    The case was initialized at 00z Oct 09, 2018 and forecast out to 90 hours. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, cycle hour, and fcst length are listed below.
```

Author: Changed, thanks!

    .. code-block:: bash

      FCST_LEN_HRS="90"
      LBC_SPEC_INTVL_HRS="6"
      DATE_FIRST_CYCL="20181009"
      DATE_LAST_CYCL="20181009"
      CYCL_HRS=( "00" ) 

    Initial condition (IC) and boundary condition (BC) files are created from GFS operational dataset in NEMSIO format.

    .. container:: sphx-glr-footer
        :class: sphx-glr-footer-example

      .. container:: sphx-glr-download sphx-glr-download-python

        :download:`Download initial condition files: 2018100900.gfs.nemsio.tar.gz <https://ufs-case-studies.s3.amazonaws.com/2018100900.gfs.nemsio.tar.gz>` 

>         :download:`Download the script for getting boundary conditions: get_hsup_bc_ic.sh <./get_hsup_bc_ic.sh>` 

Reviewer 2: ```suggestion
        
        :download:`Download the script for getting boundary conditions: get_hsup_bc_ic.sh <./get_hsup_bc_ic.sh>` 
```

    * The maximum surface wind speed at the vortex center in SRW_GFSv15p2 (90 kts) does not reach the peak intensity identified in Best Track data (140 kts).
    * The minimum sea level pressure from SRW_GFSv15p2 is 948 hpa, which is larger than Best Track data.
  

......................
Summary and Discussion
......................

The GFS.v16beta compset in MRW.v1.0 does not show improvements in simulating Hurricane Michael track and intensity when compared with the GFS.v15p2 compset. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.

> The results from SRW App v1.0 show better track and intensity in simulating Hurricane Michael when compared to the results from MRW App v1.0, possibly because the SRW App 1.0 has a newer code base. However, the SRW App forecasts have shorter duration because of limitations in the availability of lateral boundary conditions for longer lead times.

Reviewer 3: Is the initialization date the same for MRW and SRW for Hurricane Michael?

Author: The initialization date is different for MRW and SRW for Hurricane Michael due to that the boundary conditions only have 90 hours data for SRW. @climbfuji

Reviewer 3: So this means that the SRW App is initialized closer to the analysis time. This should be mentioned when saying that the SRW App produces better tracks, because the shorter the forecast time, the better the tracks will be. Same for all other hurricanes (if applicable).

Author: Yes, thanks!

Author: Yes, thanks!

    * The maximum surface wind speed at the vortex center in GFS.v16.0.10 reaches to 90 kts, which is 50 kts lower than Best Track data.
    * The minimum sea level pressure from GFS.v16.0.10 is 950 hPa, versus 919 hPa from Best Track data. 

  .. group-tab:: SRW.v1.0

>     .. figure:: images/2018Michael/tracker_Michael_srwv1.png

Reviewer 2: You can crop/trim the white spaces of the figure. It can make the figures stand out clearly. The same applies to other figures.

Author: Updated, thanks!

    * The maximum surface wind speed at the vortex center in SRW_GFSv15p2 (90 kts) does not reach the peak intensity identified in Best Track data (140 kts).
    * The minimum sea level pressure from SRW_GFSv15p2 948 hpa, which is larger than Best Track data.
  

......................
Summary and Discussion
......................

The physics compset of GFS.v16beta in MRW.v1.0 does not show improvements compared with the compset of GFS.v15p2 in simulating Hurricane Michael track and intensity. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.

> The results from SRWapp v1.0 has better track and intensity in simulating Hurricane Michael comparing with the results from MRW app v1.0; however, the SRWapp v1.0 has a newer model and shorter forcasts.

Reviewer 2: Linlin, could you please expand the explanation of the meaning of 'a newer model'? 
I like that you combined the results of MRW and SRW in the plot. But I only see SRW_GFSv15p2, not for SRW_RRFSv1alpha. Is this on purpose?

Author: RRFSv1alpha crashed, the old data seem don't work with RRFSv1alpha.

Reviewer 2: I see. Have you tried to reduce the time steps?

Author: Yes， I guess the problem is beyond the time step.

Reviewer 2: Thank you for the information Linlin! If so, I think it may worth mentioning here why RRFSv1alpha results are not mentioned here.

Author: Yes, thanks!

>     * The maximum surface wind speed at the vortex center in GFS.v16.0.10 reaches to 90 kts, which is 50 kts lower than Best Track data.

Reviewer 3: reaches 90 kts

Author: Changed, thanks!

    * The minimum sea level pressure from GFS.v16.0.10 is 950 hPa, versus 919 hPa from Best Track data. 

  .. group-tab:: SRW.v1.0

    .. figure:: images/2018Michael/tracker_Michael_srwv1.png
      :width: 400
      :align: center

      Hurricane tracks from SRW_GFSv15p2 (purple dash line), MRW_GFSv16beta (blue line), MRW_GFSv15p2 (red line),  and Best Track (black line). The dots are color coded with the vortex maximum 10-m wind speed (WS, kt). 

    * Hurricane track from SRW_GFSv15p2 is closer to the Best Track compared with MRW_GFSv15p2 and MRW_GFSv16beta. 

>     * SRW_RRFSv1alpha  can not run with the old initial and boundary conditions and crashed.

Reviewer: ```suggestion
    * Due to lack of information contained in the initial and boundary condition files for this relatively old (2018) case, the SRW App could not run with suite ``SRW_RRFSv1alpha``. 
```

Author: Changed, thanks!

    .. figure:: images/2018Michael/tracker_timeseries_Michael_srwv1.png
      :width: 1200
      :align: center

      Time series of the vortex maximum surface wind speed (WS, left panel) and minimum mean sea level pressure (MSLP, right panel)

    * The maximum surface wind speed at the vortex center in SRW_GFSv15p2 (90 kts) does not reach the peak intensity identified in Best Track data (140 kts).
    * The minimum sea level pressure from SRW_GFSv15p2 is 948 hpa, which is larger than Best Track data.
  

......................
Summary and Discussion
......................

> The physics compset of GFS.v16beta in MRW.v1.0 does not show improvements compared with the compset of GFS.v15p2 in simulating Hurricane Michael track and intensity. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.

Reviewer: ```suggestion
The GFS.v16beta compset in MRW.v1.0 does not show improvements in simulating Hurricane Michael track and intensity when compared with the GFS.v15p2 compset. The physics for MRW_GFS.v16beta was frozen in Jan 2020 for the public release of UFS MRW App version 1.0. Several major changes have been made to the GFS model after that, which include more vertical levels, several tuning parameters, and modified background diffusivity as a function of stability in the PBL scheme, etc. (`Yang 2020 <https://ufscommunity.org/wp-content/uploads/2020/10/UFS_Webnair_GFSv16_20201022_FanglinYang.pdf>`_). In GFS.v16.0.10 (GitHub tag created in Jun 2020), the simulated hurricane landfall location and hurricane intensity is closer to Best Track data compared to MRW_GFSv15p2.
```

Author: Changed, thanks!
The plotting scripts are same as the scripts used in MRW plotting. No extra scripts is needed.

> The results from SRWapp v1.0 has better track and intensity in simulating Hurricane Michael comparing with the results from MRW app v1.0; however, the SRWapp v1.0 has a newer model and shorter forcasts.

Reviewer: ```suggestion
The results from the SRW App v1.0 show better track and intensity in simulating Hurricane Michael when compared to the results from the MRW App v1.0, possibly because the SRW App v1.0 has a newer code base. However, the SRW App forecasts have shorter duration because of limitations in the availability of lateral boundary conditions for longer lead times.
```

Author: Changed, thanks!
